### ISSUE TITLE
Unlabeled data?

### ISSUE THREAD

**Issue by miladsoltany** [lucidrains/pi-GAN-pytorch#6]
Hi, 
in the paper, it's stated that the training happens on unlabeled data. My question which I can't get my head around is just how does it work?
Doesn't a NERF model give the 3d scene representation? well, it does, and only for one scene. while requiring a fair amount of data with known poses. My question is how does pi-GAN do it without labels and different images. They haven't been very thorough with the Training Details (Section 3.4 of the paper)
Thank you.
